.github/container/install-efa.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -ex
+
+# Update distro
+apt-get update
+
+# Install required packages
+apt-get install -y curl
+
+# Hack to install Amazon EFA correctly: for some reason the base docker 
+# image has incorrecly installed <libibverbs-dev>, as a result libefa* libraries
+# cannot be found under /usr/lib/x86_64-linux-gnu/
+# Temporary workaround: reinstall libibverbs-dev
+
+apt-get remove -y libibverbs-dev ibverbs-providers libibverbs1
```

Comment:
Are the existing ibverbs packages installed by the `install-ofed.sh` script?

Reply:
Per offline communication, the answer is yes, and we need to sort out a way to reconcile MOFED and EFA.

Reply:
Current solution is to install required libs from Ubuntu upstream apt repository explicitly.

Reply:
Are they different or just a dependency conflict ?

Reply:
They are different